build/media_source/system/css/fields/calendar.css

```
@@ -7,16 +7,17 @@
  }
 .calendar-container {
   float: left;
-  min-width: 160px;
+  min-width: 280px;
   padding: 0;
   list-style: none;
   border-radius: 5px;
   background-color: #ffffff !important;
   z-index: 1100 !important;
 }
 .calendar-container table {
+  margin: 0px 0px 0px 55px;
```

when the value is 0 then you should not put any units

```suggestion
  margin: 0 0 0 55px;
```

---

Would it not be better to use a `margin: 0 auto;` instead of setting a hard amount?

---

> Would it not be better to use a `margin: 0 auto;` instead of setting a hard amount?

there is an empty td element in the entire 'time' row so setting the attribute to auto won't work

---

I don't see how the empty td element would be affecting the alignment of the table itself, that td should only be affecting that row. When I submitted the original issue, I noted all of the inline edits I made and `margin: 0 auto;` on the table had no negative impact when tested. Granted, I do not know if how to script builds the popup+table would be affected by auto, so if something there does affect it, ignore my suggestion.